app/src/main/java/hr/foi/air/sportloc/view/ui/activity/AboutActivity.java

```diff
@@ -0,0 +1,30 @@
+package hr.foi.air.sportloc.view.ui.activity;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import hr.foi.air.sportloc.R;
+
+public class AboutActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_about);
+        if (getSupportActionBar() != null) {
+            getSupportActionBar().hide();
+        }
+        ButterKnife.bind(this);
+    }
+
+    @OnClick(R.id.btn_contact_us)
+    public void SendEmail() {
```

Comment:
Name of the method should start with a lowercase letter according to the Java naming convention.
```suggestion
    public void sendEmail() {
```